src/modifier.ts

```diff
@@ -97,3 +97,51 @@ export function get(
     return statModifier(subject as Effect, name);
   }
 }
+
+export type BooleanModifierValues = Partial<
```

Comment:
Is there a benefit to having them split via type here? could we not just have

```ts
export type Modifiers = Partial<{
  [modifier in BooleanModifier]: boolean;
  [modifier in ClassModifier]: Class;
  ...
}>;
```

Reply:
I think there is likely benefit to suggesting specificity, but mostly I got
red squiggles on my first try

Reply:
I can set it up like properties, though